phpunit.xml

```diff
@@ -9,7 +9,7 @@
 >
     <testsuites>
         <testsuite name="Test Suite">
-            <directory>tests/</directory>
+            <directory>tests/Annotations</directory>
```

Comment:
why this? should not be needed.

Reply:
tests for attributes do not work with PHP < 8.1

Maybe it may have the meaning to create a new version only for PHP 8.1? Swagger PHP Attributes does not work with PHP version < 8.1 properly.  

My current version tries working with all PHP versions but in CI we need to remember these restrictions. But as I say you will have many problems with version PHP 8.0. 

Some does not support features (nested attributes), for example, some dirty hack from this what I saw in PHP 8.0

```
define("CONST_OBJ_NAME", new OA\Schema(type: string))

#OA\Parameter(name: 'some_attr', schema: CONST_OBJ_NAME)
```

Reply:
Yeah attributes will only be supported with php 8.1. Please add `@requires PHP >= 8.1` as annotation to the attributes test class. This way the tests are skipped in lower php versions.

Reply:
@Tobion  done